core/src/stores/postgres.rs

```diff
@@ -2912,7 +2912,7 @@ impl Store for PostgresStore {
         let stmt = c
             .prepare(
                 "INSERT INTO data_sources_nodes \
-               (id, data_source, created, node_id, timestamp, title, mime_type, parents, document, table, folder) \
+               (id, data_source, created, node_id, timestamp, title, mime_type, parents, document, \"table\", folder) \
```

Comment:
we also have the same in the select in get_data_source_node

Reply:
fixed thanks